src/admin-views/admin-update-message.php

```diff
@@ -6,8 +6,11 @@
 
 ?>
 
-/* translators: the placeholder will contain the semantic version of The Events Calenda plugin, e.g. 1.2.3 */
-<p class="tribe-update-message"><?php echo esc_html( sprintf( __( 'You are running Version %s and deserve a hug :-)', 'the-events-calendar' ), Tribe__Events__Main::VERSION ) ); ?></p>
+
+<p class="tribe-update-message"><?php
```

Comment:
:no_entry_sign: Opening PHP tag must be on a line by itself

<sub>hash: c7399fe7401634c1c8023dd45f0d22a6</sub>